### PR TITLE
First ML model

### LINKED ISSUE #24: try to build your first ML model on the dataset

### PR DESCRIPTION
**Pull request by RicoKursidem**
Resolve #24 

I build my first ML Model with the Dataset. I used “Fr” and “n” as Input and let the Network determine the Lifetime in years in classes of: 
0, 1, 2, 3, 4 and longer than 5 Years. 

So I stayed with the classification Problem and classified the Inputs into the Years the bearing will live. 

I use 200 Epochs with a learning rate of 1e-7
At the end of the learning process, I show the two charts:
 - Accuracy over iterations 
 - average loss over iterations

### CODE REVIEW

**Comment by philippabele** [comment]
Could you remove all *.pyc files ?

https://coderwall.com/p/wrxwog/why-not-to-commit-pyc-files-into-git-and-how-to-fix-if-you-already-did

**Review by philippabele — CHANGES_REQUESTED** [review]
Please clean up the .pyc files.

**Comment by RicoKursidem** [comment]
The '.pyc' Files should be ignored from now on.

@philippabele is there a way to reference this requested change or is it enough to answer in this thread?

**Comment by philippabele** [comment]
> The '.pyc' Files should be ignored from now on.
> 
> @philippabele is there a way to reference this requested change or is it enough to answer in this thread?

@RicoKursidem For comments it is okay to answer in this thread.

Following requested changes will be pointing to lines of code. This will be different from comments but easier to handle.

File: tutorial/pytorch/1_tensors.ipynb [hunk 1]
```diff
@@ -291,12 +291,12 @@
...
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.8"
+   "version": "3.10.5 (tags/v3.10.5:f377153, Jun  6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)]"
   },
   "orig_nbformat": 4,
   "vscode": {
    "interpreter": {
-    "hash": "aee646b5f0c348debdea9b36e04014d3b501ba8b83876c774c538990ffd76616"
+    "hash": "d0b1cd08f25460b8eca4806833a13a3bdd20cef72802d0ad364ee4ea93f06117"
```
Comment (philippabele) [inline tutorial/pytorch/1_tensors.ipynb:299]: Commiting such changes makes your PR messy.

File: deep_groove_project/src/main.py [hunk 2]
```diff
@@ -1,53 +1,106 @@
...
-    ax.scatter(fr, n, lifetime)
-    ax.set_xlabel('Fr')
-    ax.set_ylabel('n')
-    ax.set_zlabel('Lifetime')
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    model = NeuralNetwork().to(device)
+    print(model)
+
+    #  Loading and preparing Dataframe
+    dataframe = pd.read_csv("../data/first_dataset.csv")
```
Comment (philippabele) [inline deep_groove_project/src/main.py]: In Contrast to your tutorial. This dataset could be committed. As long, as we do not use another solution, it is fine to commit the datasets which are unique to your project.

File: deep_groove_project/src/main.py [hunk 3]
```diff
@@ -1,53 +1,106 @@
...
+    test_dataloader = DataLoader(test_dataset, **params)
+
+    for batch in train_dataloader:
+        print(f"Batch: {batch[0]}")
+        break
+
+    learning_rate = 1e-7
+    print(f"LR:  {learning_rate}")
+
+    loss_fn = nn.CrossEntropyLoss()
```
Comment (philippabele) [inline deep_groove_project/src/main.py]: It is great to see, that you structured your code in a pure python file. The next step is to move all files belongig to the first_dataset into its own subfolder.

Feel free to find a good naming convention for the name of the subfolder like "classification_dataset1"

**Review by philippabele — CHANGES_REQUESTED** [review]
Good Job!

Please see my notes to the lines of code.

Keep moving forward :)

File: tutorial/pytorch/1_tensors.ipynb (hunk 1 above)
Comment (RicoKursidem) [reply tutorial/pytorch/1_tensors.ipynb:299]: I will watch out for these changes in the commit process and will drop the changes in the future,

File: deep_groove_project/src/main.py (hunk 3 above)
Comment (RicoKursidem) [reply deep_groove_project/src/main.py]: I added the sub_folder. Is this fine how I did it, or should I add the EDA to the classification folder.

**Review by philippabele — APPROVED** [review]
Good Job!

File: deep_groove_project/src/main.py (hunk 3 above)
Comment (philippabele) [reply deep_groove_project/src/main.py]: For now it is okay!
